metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -535,13 +560,16 @@
                          (j/execute! this stmt)))
         dynamo-delete (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
```

Comment:
can when-not be used instead?

Reply:
It could be, but I don't see that it adds anything to replace the existing line unless there's a performance difference

Reply:
I think it is in the clojure style guide, but there is a slight performance increase in that you don't have to pay for the stack call of another function. The other CMR developers will call it out (frequently).

Reply:
It's still the same number of function calls here isn't it? (when (not= ...)) vs (when-not (= ...))